serverless.yml

```diff
@@ -66,12 +66,13 @@ functions:
       - http:
           path: notifications
           method: get
-  dynamodbExample:
-    handler: bin/dynamodb-example
+  leaderboard:
+    handler: bin/leaderboard
     events:
       - http:
-          path: dynamodb-example
+          path: leaderboard
           method: get
+      - schedule: rate(1 day)
```

Comment:
for testing purposes, I put every day. We can change it to every week once we see it working in a few days